src/Mcamara/LaravelLocalization/LaravelLocalization.php

```diff
@@ -270,9 +270,6 @@ public function getLocalizedURL($locale = null, $url = null, $attributes = [], $
             $attributes = $this->extractAttributes($url, $locale);
         }
 
-        $urlQuery = parse_url($url, PHP_URL_QUERY);
-        $urlQuery = $urlQuery ? '?'.$urlQuery : '';
-
         if (empty($url)) {
             $url = $this->request->fullUrl();
             $urlQuery = parse_url($url, PHP_URL_QUERY);
```

Comment:
Now the error message from line 273 will be thrown in this line, right?

Reply:
Not really, $url had a chance to be empty at line 273, which was the reason why I was getting the warnings.
By moving it here, we're reassigning $url in case it was null. My warnings disappeared afterward.

Reply:
Also to note that $urlQuery was being reassigned in case $url was empty, so the code at lines 273&274 is redundant since we're doing the check afterwards and reassigning.